web_printscreen_zb/controllers.py

```diff
@@ -66,6 +67,14 @@ def from_data(self, fields, rows):
                         cellvalue = float(cellvalue)
                     if cellvalue is False:
                         cellvalue = None
+                    for sign in currency:
+                        if unicode(sign, "utf-8") in cellvalue:
+                            cellvalue = cellvalue.replace(
+                                unicode(sign, 'utf-8'), '')
+                            cellvalue = cellvalue.replace(
+                                ',', '').replace('.', '').replace(u'\xa0', '')
```

Comment:
Tu pourrais avoir un seul replace au lieu de 3, quelque chose comme : replace(u'(,|.|\xa0)', '')

Reply:
@yasmineelmrini based on the [2nd answer](https://stackoverflow.com/questions/3411771/multiple-character-replace-with-python).

If the report increase it is better that way.

Reply:
fair enough (y)